lib/krane/kubeclient_builder.rb

```diff
@@ -103,6 +103,14 @@ def build_storage_v1_kubeclient(context)
       )
     end
 
+    def build_scheduling_v1beta1_kubeclient(context)
```

Comment:
Is this actually necessary? I tried removing this and running tests but they keep passing; I tested with the tests modified/added in this commit:

```bash
$ alias rt="bundle exec ruby -I test"
$ rt test/integration-serial/serial_deploy_test.rb --name test_global_deploy_prune_black_box_success
$ rt test/integration/global_deploy_test.rb
$ rt test/unit/cluster_resource_discovery_test.rb
```

Am I missing anything?

Reply:
`build_scheduling_v1beta1_kubeclient` are mostly used in the tests to allow for manual interaction with resources, usually to delete them in an ensure block. They aren't being used but seemed less than ideal  to add a new kind to the fixtures but not add its helper methods.